core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,10 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def applyBoost(path: String) =
+    (path startsWith config.rootDir.getAbsolutePath) &&
```

Comment:
we have a few convenience methods around to check if a file is an ancestor of another, perhaps that might be better (and avoids using the raw `path: String` which is fragile, especially on Windows).